system/ee/ExpressionEngine/Addons/pro_search/mod.pro_search.php

```diff
@@ -896,6 +902,11 @@ public function results()
             $tagdata = ee()->extensions->call('pro_search_channel_entries');
         }
 
+        //enable anew if needed
+        if($enable_sql_caching != ee()->config->item('enable_sql_caching')) {
+            ee()->config->config['enable_sql_caching'] =  ee()->config->item('changed_enable_sql_caching');
```

Comment:
Is the logic right here?  I'm not seeing it set the changed_enable_sql_caching?

Should this actually be set to the $original_sql_caching above.. see comment above

Reply:
@TomJaeger that's what line 907 is doing. I just named it poorly and, kinda, against logic lol 

Will add some sanity to the variable key naming and update :)